### PR TITLE
[EDA] dmlWrapper is unable to process multiple primary affiliation updates per Contact

### LINKED ISSUE #1040: dmlWrapper is unable to process multiple primary affiliation updates per Contact
**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in [The Power of Us Hub](https://powerofus.force.com).
For feature requests, please use the Ideas tab in [The Power of Us Hub](https://powerofus.force.com).**

=========

**Background**: Issue was [first reported](https://powerofus.force.com/s/question/0D51E000059FuZTSA0/primary-affilations-are-not-updating-in-bulk) in the Hub on 12/19/2020. Follow up post on 12/31 confirmed another customer org was experiencing the same behavior.

=========

* **Do you want to request a *feature* or report a *bug*?**
Bug

* **What is the current behavior?**
In EDA, if the same Contact has multiple Affiliations updated to Primary at the same time, the dmlWrapper can only handle one of the updates. 

**Example/Steps to Reproduce**

First: I used data loader to insert 145 affiliations for 29 Contacts (5 Primary Affiliations a piece: Academic Program, Business Organization, Educational Institution, Sports Organization and University Department) and all Affiliations were created. It took about 5 minutes before all Primary fields on Contact were updated to reflect that insert.

Second: I went ahead and inserted another 145 non-primary affiliations for those same 29 contacts. They now have 5 non-primary affiliations and 5 primary affiliations. 

Third: I then updated the non-primary affiliations to primary. When running the New Affiliations with Contact Report (grouping by Contact ID) it shows that all 10 affiliations are now marked as Primary. The Primary fields on Contact still show the original Primary Affiliations.

* **What is the expected behavior?**
If the same Contact has multiple Affiliations updated to Primary at the same time, the dmlWrapper is able to handle all of the updates.

### PR DESCRIPTION
**Pull request by skosgi1**
# Critical Changes

# Changes
## Fix for Primary Affiliation Bulk Updates
We've fixed an issue that prevented the dmlWrapper from handling bulk updates. Now, when you update multiple Affiliations to be Primary Affiliation records, the fields on the corresponding Contact records update as expected.

# Issues Closed
Closes #1040 

# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: src/classes/AFFL_MultiRecordType_TDTM.cls [hunk 1]
```diff
@@ -109,19 +109,19 @@ public class AFFL_MultiRecordType_TDTM extends TDTM_Runnable {
                     }
 
                     // AFTER UPDATE - Runs when AFTER UPDATE Code has not already been run (checked from Recursion Flag - TDTM_ProcessControl.registeredTrigger.AFFL_MultiRecordType_TDTM_After_Update)
-					// And when BEFORE INSERT code has not already been run (checked from Recursion Flag - TDTM_ProcessControl.registeredTrigger.AFFL_MultiRecordType_TDTM_After_Update) so that values
-					// populated in BEFORE INSERT is not getting over-written in AFTER UPDATE
+					          // And when BEFORE INSERT code has not already been run (checked from Recursion Flag - TDTM_ProcessControl.registeredTrigger.AFFL_MultiRecordType_TDTM_afflMadePrimary) so that values
```
[the remark this answers is not in the document]
Comment (skosgi1) [reply src/classes/AFFL_MultiRecordType_TDTM.cls]: Thanks for the review @lifewithryan I think this is github, I tried new fixing it but it doesn't show any diff.

[the remark this answers is not in the document]
Comment (lifewithryan) [reply src/classes/AFFL_MultiRecordType_TDTM.cls]: oh sorry, I fixed it while I was reviewing it :) Thought I resolved the comment :)

[the remark this answers is not in the document]
Comment (skosgi1) [reply src/classes/AFFL_MultiRecordType_TDTM.cls]: Ah ok..Thanks you :)

**Comment by dgamboni** [comment]
Tested / Passes QE.

**Pull request merged**